service/interpreter/activityImpl_test.go

```diff
@@ -88,3 +94,141 @@ func createCommands() ([]iwfidl.TimerCommand, []iwfidl.SignalCommand, []iwfidl.I
 	}
 	return validTimerCommands, validSignalCommands, internalCommands
 }
+
+func TestComposeHttpError_LocalActivity_LongErrorResponse(t *testing.T) {
```

Comment:
Some of these test initializations are using duplicated logic as well. It'd be nice if these were consolidated. Maybe a function that takes the repeat length and returns the httpResp?

Reply:
I'll try to extract as much logic a to helper as I can

Reply:
Refactored

Reply:
Looks good